clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1943,6 +1943,13 @@ void Sema::ConstructOpenCLKernel(FunctionDecl *KernelCallerFunc,
       constructKernelName(*this, KernelCallerFunc, MC);
   StringRef KernelName(getLangOpts().SYCLUnnamedLambda ? StableName
                                                        : CalculatedName);
+  if (KernelObj->isLambda()) {
```

Comment:
LLVM coding standard says not to use braces through here.  I'd suggest removing all of them.

Reply:
Thanks @erichkeane for pointing this out. I did take a look and my understanding is that it is a good idea to leave the outer braces in there. So, I have left them in there. Seems more readable too.